examples/publish-php/publish-encrypted.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+$message = [
+    "message" => "Secret!",
+    "priority" => 5
+];
+$plaintext = json_encode($message);
+$key = deriveKey("secr3t password", "https://ntfy.sh/mysecret");
```

Comment:
i live the idea of having the fully qualified name here too as this will make the key context specific to the service and resource! That's great, as we always want unique derivations per context. But if i'm pedantic again, this will derive a different salt depending how the resource is represented (port :443 is set, trailing /, hostname aliases, missing http(s), ..)